.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+      - next
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 'lts/*'
+
+      - name: Install dependencies
+        run: |
+          npm install
+          npm run test
+          npm run build
+
+      - name: Release
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
is this a secret that have been added to the repo?

Have you seen the `permission` option? https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#permissions

example: https://github.com/Dintero/kyc-boarding/blob/master/.github/workflows/deploy-dev.yml#L14-L16

Reply:
It's automatically added to all workflows: https://docs.github.com/en/actions/security-guides/automatic-token-authentication#about-the-github_token-secret